### ISSUE TITLE
The week switching arrows are incorrecly positioned

### ISSUE THREAD

**Issue by Pastafarianist** [owncloud/calendar#917]
![image](https://cloud.githubusercontent.com/assets/1315874/9574079/d798295a-4fcb-11e5-9961-441fad59a9ff.png)

Calendar v0.7.3, ownCloud v8.1.1, Chrome v44.0.2403.157, French locale.

**Comment by georgehrke** [comment]
please use the [issue template](https://raw.githubusercontent.com/owncloud/core/master/issue_template.md) as described in our [contributing guidelines](https://github.com/owncloud/core/blob/master/CONTRIBUTING.md)

**Comment by georgehrke** [comment]
Do you use any zoom-level in chrome?

**Comment by Pastafarianist** [comment]
No. Furthermore, the problem goes away when I move the maximized Chrome window from the monitor with 1920x1080 resolution to one with 1600x1200.

**Comment by jeffrobo** [comment]
This one bugged me for a long time. Once you get enough calendars, the scrollbar appears and pushes the right-arrow down to the next line. If you want to fix it yourself, here's what I did (v8.1.1, v0.7.3):

1. Edit css/style.css:
    * Line 297 - Change 250px to 266px.
        (This widens the column wide enough so that the date and arrows are centered with a wide scrollbar.)

    * Line 377 - Comment out the entire line. /* ... */
        (Rather than floating the "Today" button to the right and pushing it back towards the left...

    * Line 378 - Change the word "right" to "left" and change 6px to 52px.
        ... this pushes it away from the other three buttons.)

    * Line 376 #datecontrol_today - Add to styles: width 49px;
        (This sets the width of the button. Firefox sets a different default width than does IE and Chrome.)

    * Line 337 #onemonthview_radio - Add to styles: width 52px;
        (This sets the width of the button. Firefox sets a different default width than does IE and Chrome.)

    * Line 333 #oneweekview_radio - Add to styles: width 46px;
        (This sets the width of the button. Firefox sets a different default width than does IE and Chrome.)

    * Line 326 #onedayview_radio - Add to styles: width 35px;
        (This sets the width of the button. Firefox sets a different default width than does IE and Chrome.)

2. Edit templates/calendar.php:

    * Line 16 - Delete the two `&nbsp;` at the end of the line.
        (Firefox uses a different size space than does IE and Chrome.)

The different browsers use different width scrollbars, but this works for IE, Chrome, and Firefox.

![cal-sb](https://cloud.githubusercontent.com/assets/6401101/9806337/64f2d67c-5811-11e5-80d4-4c5e8d715ac4.png)

**Comment by Pastafarianist** [comment]
Changed 250px to 266 px, problem solved.

![image](https://cloud.githubusercontent.com/assets/1315874/9836181/560da2fa-5a18-11e5-8fe0-4e159476abbe.png)

**Comment by georgehrke** [comment]
not an appropriate solution, right arrow button and today button are not aligned anymore.
Please use the issue template (just as asked) if you want us to fix it properly.
